_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,24 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration                                                                         | Host                | Connection type | Port/protocol                                                 |
+|---------------------------------------------------------------------------------------|---------------------|-----------------|---------------------------------------------------------------|
+| {{site.prodname}} networking                                                          | All                 | Bidirectional   | TCP 179                                                       |
+| {{site.prodname}} networking in IP-in-IP mode (default mode)                          | All                 | Bidirectional   | IP-in-IP, often represented by its protocol number `4`        |
+{%- if include.orch == "Kubernetes" %}
+| flannel networking (VXLAN)                                                            | All                 | Bidirectional   | UDP 4789                                                      |
+| etcd datastore                                                                        | etcd hosts          | Incoming        | Often TCP 2379 but varies per etcd configuration              |
+| {{site.prodname}} networking, Kubernetes API datastore, cluster of more than 50 nodes | Typha agent hosts   | Incoming        | TCP 5473 (default)                                            |
+{%- else %}
+| All                                                                                   | etcd hosts          | Incoming        | Varies per etcd configuration                                 |
+{%- endif %}
+{%- if include.orch == "Kubernetes" or include.orch == "OpenShift" %}
+| All                                                                                   | kube-apiserver host | Incoming        | Often TCP 443, but check the `port` value returned by `kubectl get svc kubernetes -o yaml` |
+{%- endif %}
+{%- if include.orch == "OpenStack" %}
+| All                                                                                   | nova-api host       | Incoming        | TCP 8775 (default)                                            |
```

Comment:
I'm not sure if it makes sense to mention this OpenStack-specific one on its own, because there are lots of other OpenStack ports that a practical deployment would need.  (https://docs.openstack.org/kilo/config-reference/content/firewalls-default-ports.html gives the idea; unfortunately I haven't been able to find where the equivalent doc is for a more recent OpenStack release - but I can keep looking if needed, so please let me know.)

I guess our options are:
1. Say that we're documenting the Calico-extra ports that are needed, in addition to whatever the relevant orchestration system might normally need.
1a. Perhaps add links out to the relevant documentation for each orchestrator, if we can find it.
2. Document the ports that a typical orchestrator deployment will need.  In the OpenStack case, this would mean adding a bunch from that doc that I linked.
3. Both (1) and (2).

WDYT?

Reply:
Hi @neiljerram OK good to know. I would suggest we go with #1 and leave it at that. Are you suggesting that the nova-api port does not fit into this category? Thanks for taking a look.

Reply:
Not sure if "this category" means

(a) the category of standard OpenStack ports that we don't need to mention in our own documentation

or 

(b) the category of Calico-extra ports that we should mention.

The nova-api port (8775) is definitely in category (a), and I don't see any real case for it to be in category (b).

Reply:
All right, great, thank you @neiljerram